proto/bazel-proto

```diff
@@ -0,0 +1 @@
+/root/.cache/bazel/_bazel_root/8f71f31e67e8d393e243d4f02ac64710/execroot/__main__
```

Comment:
What is this file?

Reply:
probably, it is cache.
i delete this and add .cache directory in ignore file.